terms.html

```diff
@@ -149,25 +151,33 @@
 
   <dd>
 The formal syntax of a <a>decentralized identifier</a>. The generic DID scheme
-begins with the prefix <code>did:</code> as defined in <a href="#did-syntax"></a>.
-Each <a>DID method</a> specification must define a specific DID
-scheme that works with that specific <a>DID method</a>. In a specific DID method
-scheme, the DID method name must follow the first colon and terminate with the
-second colon, e.g., <code>did:example:</code>
+begins with the prefix <code>did:</code> as defined in <a
+href="#did-syntax"></a>. Each <a>DID method</a> specification defines a specific
+DID scheme that works with that specific <a>DID method</a>. In a specific DID
+method scheme, the DID method name follows the first colon and terminates with
+the second colon, e.g., <code>did:example:</code>
   </dd>
 
   <dt><dfn data-lt="DID subjects">DID subject</dfn></dt>
 
   <dd>
-The entity identified by a <a>DID</a> and described by a <a>DID document</a>. A
-<a>DID</a> has exactly one DID subject. Anything can be a DID subject: person,
-group, organization, physical thing, digital thing, logical thing, etc.
+The entity identified by a <a>DID</a> and described by a <a>DID document</a>.
+Anything can be a DID subject: person, group, organization, physical thing,
+digital thing, logical thing, etc.
+  </dd>
+
+  <dt><dfn data-lt="service">DID services</dfn></dt>
+  <dd>
+Means of communicating or interacting with the <a>DID subject</a> or
+associated entities via one or more <a>service endpoints</a>.
+Examples include discovery services, agent services, social networking
+services, file storage services, and verifiable credential repository services.
```

Comment:
```suggestion
Examples of services include discovery, agent, social networking, file storage,
and verifiable credential repository services.
```

Reply:
Could also include "negotiator", "mediator", and/or "proxy" as examples of services, to correspond with recent updates in the Service Privacy section